commands/command_pointer.go

```diff
@@ -85,7 +85,7 @@ func pointerCommand(cmd *cobra.Command, args []string) {
 		fmt.Printf("Pointer from %s\n\n", pointerName)
 
 		if err != nil {
-			Error(err.Error())
+			Error("Not a valid Git LFS pointer file.")
```

Comment:
Isn't this losing internal error details? That message is taking from the `lfs.NotAPointerError` var thats' removed in lfs/pointer.go.